### ISSUE TITLE
Clarify that not all exempt content fails conformance

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#2849]
### Section

https://www.w3.org/TR/epub-a11y-exemption/#sec-evaluator-name

### Describe the problem

Looking at the exemption note today I spotted this:

> When a jurisdiction provides specific legal exemptions for [EPUB publications](https://www.w3.org/TR/epub-33/#dfn-epub-publication), [EPUB creators](https://www.w3.org/TR/epub-33/#dfn-epub-creator) whose content fails to meet the requirements of the EPUB Accessibility standard [[epub-a11y](https://www.w3.org/TR/epub-a11y-exemption/#bib-epub-a11y)] (or any other standard), can use [the exemption property](https://www.w3.org/TR/epub-a11y-exemption/#exemption-property) to indicate when their publications fall under one of these provisions.

This isn't fully accurate, since a publication could claim level A conformance yet still need an exemption from the EAA which requires AA conformance.

### Describe the fix or new feature you propose

Clarify that any publication that conforms below the necessary minimum can also claim exemptions.

(And write out all the "epub creators" while in there. Also fix the "sec-evaluator-name" id on the section as it looks like a clear case of a copy/paste.)
